AFMReader/jpk.py

```diff
@@ -35,7 +35,62 @@ def _jpk_pixel_to_nm_scaling(tiff_page: tifffile.tifffile.TiffPage) -> float:
     return px_to_nm * 1e9
 
 
-# pylint: disable=too-many-locals
+def _get_z_scaling(tif: tifffile.tifffile, channel_idx: int) -> tuple[float, float]:
+    """
+    Extract the z scaling factor and offset for a JPK image channel.
+
+    Determined using JPKImageSpec.txt Version: 2.0:3fffffffffff supplied with JPK instrument.
+
+    Parameters
+    ----------
+    tif : tifffile.tifffile
+        A tiff file of .jpk images.
+    channel_idx : int
+        Numerical channel identifier used to navigate the tifffile pages.
+
+    Returns
+    -------
+    tuple[float, float]
+        A tuple contains values used to scale and offset raw data.
+    """
+    n_slots = tif.pages[channel_idx].tags["32896"].value
+    default_slot = tif.pages[channel_idx].tags["32897"]
+
+    # Create a dictionary of list for the differnt slots
+    slots = {slot: [] for slot in range(n_slots)}
```

Comment:
An educated guess after reading the code is that typehint required here to placate pre-commit is...

```suggestion
    slots dict[int, list[str]] = {slot: [] for slot in range(n_slots)}
```

Reply:
I've tried a few things and can't get it to go, this throws up a syntax error.

Reply:
A colon fixed it!